packages/flutter/pubspec.yaml

```diff
@@ -19,5 +22,13 @@ dependencies:
   newton:
     path: ../newton
 
+dev_dependencies:
+  flx:
+    path: ../flx
+  flutter_test:
+    path: ../flutter_test
+  stocks:
+    path: ../../examples/stocks
```

Comment:
yup, will do. Sorry, I hadn't really finished this yet, I was mostly testing to see what Travis would do. :-)